document/FileFormat.mdk

```diff
@@ -1656,8 +1699,9 @@ or include them in bug counts.
 ~
 
 If present, this property conveys the reason or reasons that the result has been suppressed.
-In this version of the SARIF standard, the only supported reason for suppressing a result is that
-the developer has suppressed it in the source code (see &sect;[#result-suppressionStates-suppressedInSource]).
+In this version of the SARIF standard, the only supported reasons for suppressing a result is that
+the developer has suppressed it in the source code (see &sect;[#result-suppressionStates-suppressedInSource])
+or that it is marked as "suppressed" an external database (see &sect;[#result-suppressionStates-suppressedExternally]).
```

Comment:
fixed, fixed.

---
In reply to: [75183149](https://github.com/sarif-standard/sarif-spec/pull/256#discussion_r75183149) [](ancestors = 75183149)